service/src/main/java/eu/europa/ec/fisheries/ers/fa/dao/FishingActivityDao.java

```diff
@@ -108,7 +108,6 @@ public int getPreviousFishingActivityId(int fishingActivityId, String activityTy
      * @param fishingActivityId
      * @param activityTypeCode
      * @param activityCalculatedStartTime
-     * @return
```

Comment:
Why remove the return? Wouldn't it be better to add some documentation? Either we try to keep it up to date or we remove it completely.

Reply:
If there is relevant info that is not expressed via method/parameter names&types I think it is great if that was added as javadoc. This is out of scope for this commit which was just about getting rid of build warnings without loosing any functionality or information